gradle.properties

```diff
@@ -9,8 +9,8 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx1024m
-
+#org.gradle.jvmargs=-Xmx1024m
```

Comment:
Prefer not to comment out code--remove/update it instead.

Reply:
removed the commented code

Reply:
done